halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/ConsulClientService.java

```diff
@@ -117,6 +117,7 @@ public static String consulClientService(String serviceName) {
     Boolean sidecar = true;
     Integer targetSize = 1;
     Map<String, String> env = new HashMap<>();
+    Map<String, String> volumeMounts = new HashMap<>();
```

Comment:
How do these work for non k8s deployments? I do like the idea, just making sure they work across platforms.